java/flight/src/main/java/org/apache/arrow/flight/FlightInfo.java

```diff
@@ -35,13 +35,24 @@
 import com.google.common.collect.ImmutableList;
 import com.google.protobuf.ByteString;
 
+/**
+ * A POJO representation of a FlightInfo, metadata associated with a set of data records.
+ */
 public class FlightInfo {
   private Schema schema;
   private FlightDescriptor descriptor;
   private List<FlightEndpoint> endpoints;
   private final long bytes;
   private final long records;
 
+  /**
+   * Constructs a new instance
```

Comment:
add a new line here to be consistent. Can there be a checkstyle rule for that?

Reply:
fixed in pending commit.